### PR TITLE
Homework 2 of Stelian Todorichkov 18

### PR DESCRIPTION
**Pull request by StelianRBG**

### CODE REVIEW

**Review by Alexander1022 — APPROVED** [review]
I see your improvements! I like the way you are organizing the data with dict of lists! 🙌

File: homework_02/stelian_todorichkov_18/views.py [hunk 1]
```diff
@@ -38,14 +38,8 @@ def board(request):
...
     # sort list by time
-    for i in range(0, len(list)):
-        for k in range(i+1, len(list)):
-            
-            if list[i]['time'] > list[k]['time']:
-                dict = list[i]
-                list[i] = list[k]
-                list[k] = dict
-
+    list = sorted(list, key = lambda i: i['time'])
```
Comment (Alexander1022) [inline homework_02/stelian_todorichkov_18/views.py:41]: A shorter way to sort the elements, I like it!

**Review by Alexander1022 — APPROVED** [review]

File: homework_02/stelian_todorichkov_18/static/style.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+table tbody td {
+	color: #636363;
+	border: 1px solid #dddfe1;
+}
+table tbody tr {
+	background-color: #f9fafb;
+}
+table tbody tr:nth-child(odd) {
+	background-color: #ffffff;
+}
```
Comment (marti456) [inline homework_02/stelian_todorichkov_18/static/style.css:51]: Good CSS for table

File: homework_02/stelian_todorichkov_18/templates/board.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                {% for status in statuses %}
+                    {% if forloop.counter == forloop.parentloop.counter %}
+                        <td>{{ status }}</td>
+                    {% endif %}
+                {% endfor %}
+            </tr>
+            {% endfor %}    
+        </tbody>
+    </table>
+</html>
```
[the remark this answers is not in the document]
Comment (marti456) [reply homework_02/stelian_todorichkov_18/templates/board.html:36]: Frontend is good enought

File: homework_02/stelian_todorichkov_18/views.py (hunk 1 above)
Comment (marti456) [reply homework_02/stelian_todorichkov_18/views.py:41]: The logic is good and the code is readable.

File: homework_02/stelian_todorichkov_18/views.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                el['time'] = data['attributes']['departure_time']
+
+    # sort list by time
+    list = sorted(list, key = lambda i: i['time'])
+    
+    # format time 
+    for el in list:
+        el['time'] = datetime.fromisoformat(el['time']).strftime('%I:%M %p')
+        
+    cur_time = datetime.now().astimezone(timezone('America/New_York')).strftime('%I:%M:%S %p')
```
Comment (marti456) [inline homework_02/stelian_todorichkov_18/views.py:47]: Good way to transform types of the time variable

File: homework_02/stelian_todorichkov_18/views.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        'time' : cur_time.strftime("%H:%M"),
+
+        'data' : response_json,
+        'routes' : routes,
+        'statuses' : vehicle_status,
+    }
+
+    return render(request, "board.html", context)
+
+
```
[the remark this answers is not in the document]
Comment (marti456) [reply homework_02/stelian_todorichkov_18/views.py:56]: +1 good backend

**Review by marti456 — APPROVED** [review]
Good, readable and working code

**Pull request merged**